readme.md

```diff
@@ -3,15 +3,15 @@
 ## Learning Objectives
 
 - Give an example of a problem solved by Flexbox.
-- Contrast flex containers and flex items.
+- Contrast flex containers and flex-items.
```

Comment:
Do we need the hyphen? Unless flex items is usually written this way, should make the way we write "flex containers" and "flex items" consistent.